src/adagrams.js

```diff
@@ -1,15 +1,158 @@
-export const drawLetters = () => {
-  // Implement this method for wave 1
-};
+class Adagrams {
+  drawLetters = () => {
+    // Implement this method for wave 1
+    const poolOfLettersDict = {
+      A: 9,
+      B: 2,
+      C: 2,
+      D: 4,
+      E: 12,
+      F: 2,
+      G: 3,
+      H: 2,
+      I: 9,
+      J: 1,
+      K: 1,
+      L: 4,
+      M: 2,
+      N: 6,
+      O: 8,
+      P: 2,
+      Q: 1,
+      R: 6,
+      S: 4,
+      T: 6,
+      U: 4,
+      V: 2,
+      W: 2,
+      X: 1,
+      Y: 2,
+      Z: 1,
+    };
+    const lettersInHand = new Array();
+    let counter = 0;
+    const characters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+    while (counter < 10) {
```

Comment:
Instead of using a counter that you need to keep track of yourself, you can use the length of `lettersInHand`. If the length is less than 10, keep going.